latch_cli/services/init/init.py

```diff
@@ -1,142 +1,172 @@
 """Service to initialize boilerplate."""
 
+import json
 import os
 import re
 import shutil
 from enum import Enum, auto
+from importlib.metadata import version
 from pathlib import Path
 from textwrap import dedent
-from typing import Optional
+from typing import Callable, Optional
+
+import click
+
+from latch_cli.tui import select_tui
 
 
 class _Templates(Enum):
-    default = auto()
+    empty = auto()
+    subprocess = auto()
     r = auto()
     conda = auto()
 
 
-def init(pkg_name: Path, template: Optional[str] = None):
+def init(pkg_name: str, expose_dockerfile: bool = True) -> bool:
```

Comment:
`expose_dockerfile` -- to be used in next PR but I did this here when removing template.